package.json

```diff
@@ -1,35 +1,16 @@
 {
-  "name": "web-extension-starter",
+  "name": "@blockstack/app",
+  "description": "The Blockstack App is a browser extension for interacting with Blockstack apps.",
   "version": "1.0.0",
-  "description": "A web browser extension starter - based on Typescript, React, Redux, Styled-Component, Webpack, and more. Runs on Chrome and Firefox.",
-  "main": "index.js",
-  "scripts": {
```

Comment:
Sorry for pending rant, but:

Can we please 🙏not order our package.json in alphabetical order? This is the IDE-feature autoformat wild west: alphabetical order is, by far, the most least optimal way of ordering this file imo.

The most important and frequent thing I go to check in the package.json, is what scripts I can run. Now they're at the bottom for no good reason. 

When you run `npm init` it generates a file where name, main entry file and scripts are at the top, and, I would argue, with good reason.

Reply:
I am fine with this, sure. I use a package called [`fixpack`](https://github.com/henrikjoreteg/fixpack) to automate it and lint it. Happy to move the scripts to the top.

Reply:
I'm happy to have some logic behind ordering, too.

Reply:
Interesting, haven't seen this package, and thank would really appreciate that.

> A package.json file scrubber for the truly insane. 

🤔😂

I'm cool with linting it, just arguing scripts are amongst the most important content of the file.

Reply:
That said, adding a whole new configuration file to a repo, to lint and sort one configuration file, begs the question to me... what lints the linting configuration file?

Reply:
> That said, adding a whole new configuration file to a repo, to lint and sort one configuration file, begs the question to me... what lints the linting configuration file?

It's not added to the file, it's just a one time thing generally -- `npx fixpack`. I just prefer to have some logic and consistency, I don't really care what that is :)

Reply:
Sure, but [if we wanna add scripts to the top, consistently...](https://github.com/henrikjoreteg/fixpack#configuration).

Reply:
oh nice, yeah let's do that